### PR TITLE
HPCC-29072 Bump vcpkg versions for v9 release

### PR DESCRIPTION
**Pull request by GordonSmith**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: cmake_modules/commonSetup.cmake [hunk 1]
```diff
@@ -743,7 +743,7 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
       ENDIF(CONTAINERIZED)
 
       IF (USE_ICU)
-        find_package(ICU COMPONENTS data i18n io tu uc)
+        find_package(ICU COMPONENTS data i18n io uc)
```
Comment (ghalliday) [inline cmake_modules/commonSetup.cmake:746]: What does the tu component correspond to?

File: plugins/h3/h3.cpp [hunk 2]
```diff
@@ -66,269 +66,297 @@ static void toSetOf(bool &__isAllResult, size32_t &__lenResult, void *&__result,
 namespace h3
 {
 
-//--------------------------------------------------------------------------------
-//                           ECL SERVICE ENTRYPOINTS
-//--------------------------------------------------------------------------------
+    //--------------------------------------------------------------------------------
```
Comment (ghalliday) [inline plugins/h3/h3.cpp:69]: The contents of namespaces aren't generally indented.

**Review by ghalliday — COMMENTED** [review]
Looks reasonable.  One comment, one question.

File: plugins/h3/h3.cpp (hunk 2 above)
Comment (GordonSmith) [reply plugins/h3/h3.cpp:110]: Whoops - auto format I am afraid - I will fix.

**Comment by GordonSmith** [comment]
Squashed and rebased

**Comment by GordonSmith** [comment]
@ghalliday bump

**Comment by GordonSmith** [comment]
Closing until 9.2.x

**Pull request closed without merging**